app/src/main/res/layout/activity_history.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout
+        xmlns:android="http://schemas.android.com/apk/res/android"
+        xmlns:tools="http://schemas.android.com/tools"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:background="@android:color/background_dark"
+        android:orientation="vertical"
+        tools:context=".MainActivity">
+    <TextView
+            android:id="@+id/history_text"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:textColor="#60D8A7"
+            android:gravity="end"
+            android:ellipsize="end"
+            android:textSize="30sp"
+            android:textAppearance="@style/TextAppearance.AppCompat.Body1"
+            android:fontFamily="sans-serif-medium" android:textStyle="bold"
```

Comment:
before pushing code use CTRL+L, to format all text, cause each parameter should be one under another.